distmv/normal.go

```diff
@@ -75,11 +75,17 @@ func NewNormal(mu []float64, sigma mat64.Symmetric, src *rand.Rand) (*Normal, bo
 // ConditionNormal returns {nil, false} if there is a failure during the update.
 // Mathematically this is impossible, but can occur with finite precision arithmetic.
 func (n *Normal) ConditionNormal(observed []int, values []float64, src *rand.Rand) (*Normal, bool) {
+	if len(observed) == 0 {
+		panic("normal: no observed values")
```

Comment:
That's an interesting observation. Number differences between ASE and BSE/AuSE are fun. An example of this is that we call an organisation plural, e.g. "gonum are developing a suite of packages" on the basis that gonum are a group of people which is plural.